librz/include/rz_util/rz_print.h

```diff
@@ -26,7 +26,6 @@ extern "C" {
 #define RZ_PRINT_FLAGS_COMPACT  0x00000800
 #define RZ_PRINT_FLAGS_NONHEX   0x00001000
 #define RZ_PRINT_FLAGS_SECSUB   0x00002000
-#define RZ_PRINT_FLAGS_RAINBOW  0x00004000
```

Comment:
do we want to move up all the various flags by one?

Reply:
I don't think it's necessary. If there will be need for a new flag, we can insert it in this "slot"